google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITGrpcTest.java

```diff
@@ -18,24 +18,29 @@
 
 import static com.google.common.truth.Truth.assertThat;
 
+import com.google.api.gax.paging.Page;
 import com.google.cloud.NoCredentials;
+import com.google.cloud.storage.Blob;
 import com.google.cloud.storage.Bucket;
+import com.google.cloud.storage.BucketFixture;
 import com.google.cloud.storage.BucketInfo;
 import com.google.cloud.storage.StorageFixture;
 import com.google.cloud.storage.StorageOptions;
+import com.google.cloud.storage.conformance.retry.CleanupStrategy;
 import com.google.cloud.storage.conformance.retry.TestBench;
 import com.google.cloud.storage.testing.RemoteStorageHelper;
+import com.google.common.collect.ImmutableList;
+import java.util.stream.StreamSupport;
 import org.junit.ClassRule;
-import org.junit.Rule;
 import org.junit.Test;
 
 public final class ITGrpcTest {
-  @ClassRule
+  @ClassRule(order = 1)
   public static final TestBench TEST_BENCH =
       TestBench.newBuilder().setContainerName("it-grpc").setDockerImageTag("v0.26.0").build();
 
-  @Rule
-  public final StorageFixture storageFixture =
+  @ClassRule(order = 2)
```

Comment:
this is more a question than a comment, what is the benefit to using ClassRule over Rule here?

Reply:
`@ClassRule` will bind to the lifecycle of the class, whereas `@Rule` will bind to the lifecycle of the test.

In this case, the storage client can live as long as the testbench since we're not testing anything on the construction/setup of the storage instance.